docker-compose.restore.yml

```diff
@@ -1,19 +1,13 @@
-version: '3'
 services:
   borgmatic:
     container_name: borg-restore
-
     cap_add:
       - SYS_ADMIN
-
     volumes:
       - /host/mount/location:/restore
-
     security_opt:
       - apparmor:unconfined
       - label:disable
-
     devices:
       - /dev/fuse:/dev/fuse
-
-    command: /bin/sh
+    command: /bin/sh
```

Comment:
We have bash available to us now just FYI. This may allow for more advanced commands over Ash.

Reply:
@modem7 What else should I adjust in the file - I have actually only removed empty paragraphs 🧐😅

Reply:
@Psycho0verload This is what happens when you review things just after you've gotten up, but before coffee. I'm a muppet. Let's leave as is, and we can improve it later down the line, it's beyond the scope of this PR.

Reply:
Test